tests/installation/check_releasenotes.pm

```diff
@@ -0,0 +1,30 @@
+# SUSE's openQA tests
+#
+# Copyright © 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Check releasenotes during installation
```

Comment:
Well, we already check the release notes during installation. This test is doing something more specific, e.g. check release notes location or origin. This should also be reflected in the filename. Otherwise hard to understand why we have "installation/releasenotes" and also "installation/check_releasenotes"

Reply:
Yup, will change